Code/GraphMol/MolPickler.cpp

```diff
@@ -929,14 +929,25 @@ void MolPickler::_pickle(const ROMol *mol, std::ostream &ss,
     }
   }
 
-  // pickle the conformations
-  streamWrite(ss, BEGINCONFS);
-  tmpInt = static_cast<int32_t>(mol->getNumConformers());
-  streamWrite(ss, tmpInt);
+  if (propertyFlags & PicklerOps::ConfDouble) {
```

Comment:
If you can think of something better than ConfDouble :)

ConfsAsDouble?

Reply:
@bp-kelley `CoordAsDouble`?

Reply:
How about `CoordsAsDouble`?

Reply:
I'd go with `CoordsAsDouble`